### ISSUE TITLE
Indentation within WITH ... ENDWITH

### ISSUE THREAD

**Issue by AlexMFrank** [SAP/abap-cleaner#98]
Hi,

The code between WITH ... ENDWITH currently doesn't get indented but is aligned with the WITH and ENDWITH keywords. I'd expect that it would be indented. Between SELECT ... ENDSELECT it is working fine.

Formatted with Pretty Printer (shift+F1):
```ABAP
CLASS zcl_test IMPLEMENTATION.
  METHOD if_oo_adt_classrun~main.
    WITH +a AS ( SELECT * FROM I_Plant ) SELECT * FROM +a INTO @FINAL(row).
      out->write( row ).
    ENDWITH.

    SELECT * FROM I_Plant INTO @FINAL(row2).
      out->write( row2 ).
    ENDSELECT.
  ENDMETHOD.
ENDCLASS.
```
Formatted with ABAP Cleaner (ctrl+4):
```ABAP
CLASS zcl_test IMPLEMENTATION.
  METHOD if_oo_adt_classrun~main.
    WITH +a AS ( SELECT * FROM I_Plant ) SELECT * FROM +a INTO @FINAL(row).
    out->write( row ).
    ENDWITH.

    SELECT * FROM I_Plant INTO @FINAL(row2).
      out->write( row2 ).
    ENDSELECT.
  ENDMETHOD.
ENDCLASS.
```
I've used the latest ABAP Cleaner version 1.5.2 with the default profile.

**Comment by jmgrassau** [comment]
Hi Alex, 

thanks for reporting this! The question whether or not ```WITH``` starts a loop (and therefore requires ```ENDWITH```), is not a trivial one, but this issue should be fixed with the next release.

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Alex, 

thanks again for opening this issue! This should now be fixed with version 1.5.3, which was just released. 

Kind regards,
Jörg-Michael
